### DISCUSSION TITLE
Is it possible to have two different published feeds for two separate locations?

### DISCUSSION THREAD

**Discussion by tamaramzr** [sheltermanager/asm3#1407] [Q&A / Help] [answered]
We have two parts to our rescue – a shelter that needs a feed for found animals and our main foster-based rescue that needs a feed for adoptable animals.

Under publishing options, only one location can be selected. Is there any other way we can have two so that we can show found animals and adoptable animals on separate areas of our site?

**Comment by Tetzor** [comment, marked as answer]
Hi,

When you say "Under publishing options, only one location can be selected" do you mean internal locations? If so you can create more in Settings->Lookup data->Internal Locations.

You can then use "SHELTERLOCATIONNAME" to filter animals in different places on your website. 

https://sheltermanager.com/repo/asm3_help/website.html
https://sheltermanager.com/repo/asm3_help/service.html#filters

If you are marking the animals as held, you could also use this to separate them on your website.

https://sheltermanager.com/repo/asm3_help/service.html#html-held-animals

Kind regards,

Jon
